command/agent/kvs_endpoint.go

```diff
@@ -91,15 +91,15 @@ func (s *HTTPServer) KVSGet(resp http.ResponseWriter, req *http.Request, args *s
 
 // KVSGetKeys handles a GET request for keys
 func (s *HTTPServer) KVSGetKeys(resp http.ResponseWriter, req *http.Request, args *structs.KeyRequest) (interface{}, error) {
-	// Check for a seperator, due to historic spelling error,
+	// Check for a separator, due to historic spelling error,
 	// we now are forced to check for both spellings
 	var sep string
 	params := req.URL.Query()
 	if _, ok := params["seperator"]; ok {
 		sep = params.Get("seperator")
 	}
-	if _, ok := params["separator"]; ok {
-		sep = params.Get("separator")
+	if _, ok := params["seperator"]; ok {
+		sep = params.Get("seperator")
```

Comment:
That's the outdated diff (that is, it's your first diff).  The spelling in master is `seperator`: https://github.com/hashicorp/consul/blob/e347f20790b81a7a64bff05756900e9294f46892/command/agent/kvs_endpoint.go#L98